Cargo.lock

```diff
@@ -127,9 +127,9 @@ checksum = "23b62fc65de8e4e7f52534fb52b0f3ed04746ae267519eef2a83941e8085068b"
 
 [[package]]
 name = "async-trait"
-version = "0.1.42"
+version = "0.1.48"
```

Comment:
This bumps more than target-lexicon. Could the other updates be split into a new PR.

Reply:
Yeah, just ran `cargo update`. I'm happy to do so, or revert the updates unrelated to target-lexicon. But in general patch bumps shouldn't bring in breaking changes (that's unfortunately not the case for this target-lexicon patch bump, which brings a breaking API change), what advantages do you see in doing so?

Reply:
For `0.x.y`, `y` bumps may still add features. Only breaking changes are not allowed.

Reply:
I've opened https://github.com/bytecodealliance/target-lexicon/issues/71 to discuss this, since in fact I think that adding the new enum variant is a breaking API change (this may break users' match statements, since the enum isn't marked non-exhaustive), but it's subtle. In any case, I've reverted the other packages updates.

Reply:
I've bumped target-lexicon to 0.12.0, which is the new pre-major release for this new enum variant.